integration_tests/no_concurrency/sdk/requirements_test.py

```diff
@@ -46,44 +46,14 @@ def _install_transformers_package():
     _run_shell_command(f"{sys.executable} -m pip install transformers")
 
 
-def _check_infer_requirements(transformers_exists: bool):
-    """Runs our supplied `infer_requirements` method and checks for the presence of `transformers`."""
-    inferred_reqs = infer_requirements()
-    if transformers_exists:
-        assert any(req_str.startswith("transformers") for req_str in inferred_reqs)
-    else:
-        assert all(not req_str.startswith("transformers") for req_str in inferred_reqs)
-
-
-@op()
+@op(requirements=[])
```

Comment:
Same comment about adding `requirements.txt` to `integration_tests/no_concurrency/sdk`.